### PR TITLE
feat: add back our jobs page

### PR DESCRIPTION
**Pull request by eseidel**
Hiring for two roles:
* Systems Engineer
* Go-to-market lead

### CODE REVIEW

File: src/pages/jobs.md [hunk 1]
```diff
@@ -4,9 +4,41 @@ title: Jobs
...
 # Work at Shorebird
 
-Shorebird is not actively hiring at this time.
+Shorebird is building "the Flutter company". Eric founded and led Flutter while
+at Google, and now together we are building a company to take Flutter beyond
+where Google can. We're on a mission to help the world "stop writing everything
+twice".
+
+We need smart, hard-working people, who want to work on start-up problems. We
+have thousands of monthly active users, and more to do our current team can keep
```
Comment (felangel) [inline src/pages/jobs.md:22]:
```suggestion
We need smart, hard-working people, who want to work on start-up problems. We
have thousands of monthly active users, and more to do than our current team can keep
```

File: src/pages/jobs.md [hunk 2]
```diff
@@ -4,9 +4,41 @@ title: Jobs
 description: Shorebird openings
 ---
 
+We're hiring
+
+- [Systems Engineer](#systems-engineer)
+- [Go-to-market lead](#go-to-market-lead)
```
Comment (bryanoltman) [inline src/pages/jobs.md:10]:
```suggestion
- [Go-to-market Lead](#go-to-market-lead)
```

File: src/pages/jobs.md [hunk 3]
```diff
@@ -4,9 +4,41 @@ title: Jobs
...
+
 # Work at Shorebird
 
-Shorebird is not actively hiring at this time.
+Shorebird is building "the Flutter company". Eric founded and led Flutter while
+at Google, and now together we are building a company to take Flutter beyond
+where Google can. We're on a mission to help the world "stop writing everything
+twice".
+
+We need smart, hard-working people, who want to work on start-up problems. We
```
Comment (bryanoltman) [inline src/pages/jobs.md:21]:
```suggestion
We need smart, hard-working people who want to work on start-up problems. We
```

File: src/pages/jobs.md [hunk 4]
```diff
@@ -4,9 +4,41 @@ title: Jobs
...
+have thousands of monthly active users, and more to do our current team can keep
+up with. No two days will be the same. You need to be strongly autonomous and
+self-directing.
+
+Our engineering work is _building_ Flutter, but don't really
+get to use it much. Shorebird engineers should have some familiarity with Dart
```
Comment (bryanoltman) [inline src/pages/jobs.md:27]:
```suggestion
Our engineering work is _building_ Flutter itself, not _with_ Flutter.
Shorebird engineers should have some familiarity with Dart
```

File: src/pages/jobs.md [hunk 5]
```diff
@@ -4,9 +4,41 @@ title: Jobs
...
+and Flutter is helpful, but mostly we need people who want to talk to customers
```
Comment (felangel) [inline src/pages/jobs.md:28]:
```suggestion
and Flutter, but mostly we need people who want to talk to customers
```

File: src/pages/jobs.md [hunk 6]
```diff
@@ -4,9 +4,41 @@ title: Jobs
...
+and then build things to solve their problems (whether by writing code, docs, or
+whatever).
+
+We hang out with our customers all day, every day, on Discord. Most of our
+stack is written in Dart, but we also have bits of JSX, Rust and C++ thrown in.
```
Comment (bryanoltman) [inline src/pages/jobs.md:33]:
```suggestion
stack is written in Dart, but we also have bits of Typescript, Rust, and C++ thrown in.
```

Comment (felangel) [inline src/pages/jobs.md:33]:
```suggestion
stack is written in Dart, but we also have bits of TypeScript, Rust and C++ thrown in.
```

File: src/pages/jobs.md [hunk 7]
```diff
@@ -4,9 +4,41 @@ title: Jobs
...
+Most of our code is open source. We run a small cloud, serving 200+ requests per
+second across several providers to millions of users around the globe.
+
+We raised a seed round from great investors and have hundreds of paying
+customers, 10k+ monthly revenue and 2+ years of runway. We're happy to put you
```
Comment (bryanoltman) [inline src/pages/jobs.md:38]:
```suggestion
customers, $10k+ monthly revenue and 2+ years of runway. We're happy to put you
```

File: src/pages/jobs.md [hunk 8]
```diff
@@ -40,13 +72,154 @@ all remote, distributed team.
...
+on the C++/C/Rust code that powers Dart VM and compiler, Flutter’s engine and
+Shorebird’s updater. This will include working on performance at these layers
+and working with the rest of the team to maintain our forks of millions of lines
+of C++ code.
+
+You will be responsible for many design decisions within the product, and will
+be expected to be able to make tradeoffs between different approaches. You will
+be expected to be able to work within a distributed team of engineers to build a
+product that is easy to use, performant, secure and delightful to developers.
+That starts of course by building something we ourselves want to use.
```
Comment (bryanoltman) [inline src/pages/jobs.md:100]: Maybe this could be a bulleted list?

File: src/pages/jobs.md [hunk 9]
```diff
@@ -40,13 +72,154 @@ all remote, distributed team.
...
+
+## Location
+
+We're all-remote, current team members are located in California, Illinois and
+New York (GMT-8 through GMT-5). Will consider GMT-10 through GMT-3. We
+communicate exclusively through Discord (both in public and private) including
+often leaving video calls open in the background while we work independently.
+
+We get together 3-4x per year as a company, all flying to some US city.
```
Comment (bryanoltman) [inline src/pages/jobs.md:109]:
```suggestion
We're fully remote. Current team members are located in California, Illinois, and
New York (GMT-8 through GMT-5). We are happy to consider anyone in GMT-10 through GMT-3. We
communicate exclusively through Discord (both in public and private), including
often leaving video calls open in the background while we work independently.

We get together 3-4x per year as a company, all flying to some US city.
```

File: src/pages/jobs.md [hunk 10]
```diff
@@ -40,13 +72,154 @@ all remote, distributed team.
...
+
+## Compensation
+
+180-210k USD salary, 1.5% equity, and
+[benefits](https://handbook.shorebird.dev/benefits/).
+
+## Requirements
```
Comment (bryanoltman) [inline src/pages/jobs.md:116]: This restates a bunch of stuff in the previous sections. Maybe we could roll the responsibilities and location sections into this?

**Review by felangel — APPROVED** [review]

File: src/pages/jobs.md (hunk 10 above)
Comment (eseidel) [reply src/pages/jobs.md]: Addressed

File: src/pages/jobs.md (hunk 8 above)
Comment (eseidel) [reply src/pages/jobs.md]: Done.

**Pull request merged**